data/campaigns/World_Conquest/Modding.md

```diff
@@ -0,0 +1,490 @@
+        # as compensation if the unit type is deemed weak (such as goblin spearman).
+        [trait_extra]
+            types=Walking Corpse
+            {WORLD_CONQUEST_II_TRAIT_LEGENDARY_ZOMBIE}
+        [/trait_extra]
+        [trait_extra]
+            types=Goblin Spearman
+            {WORLD_CONQUEST_II_TRAIT_LEGENDARY_GOBLIN}
+        [/trait_extra]
+        [trait_extra]
+            types=Orcish Assassin,Young Ogre,Ruffian,Woodsman
+            {WORLD_CONQUEST_II_TRAIT_EPIC}
+        [/trait_extra]
+        [trait_extra]
+            types=Ghoul,Poacher,Thief,Footpad,Saurian Skirmisher,Vampire Bat,Peasant,Dune Herbalist
+            {WORLD_CONQUEST_II_TRAIT_EXPERT}
+        [/trait_extra]
+        [trait_extra]
+            types=Elvish Archer,Elvish Shaman,Elvish Scout,Elvish Fighter
+            {TRAIT_DEXTROUS}
+        [/trait_extra]
+        [trait_extra]
+            types=Dwarvish Guardsman,Dwarvish Ulfserker,Dwarvish Thunderer,Dwarvish Scout
+            {TRAIT_HEALTHY}
+        [/trait_extra]
+        [trait_extra]
+            types=Naga Fighter,Wolf Rider,Orcish Grunt,Drake Glider,Dune Rover,Dune Rider
+            {TRAIT_STRONG}
+        [/trait_extra]
+        [trait_extra]
+            types=Spearman,Fencer,Cavalryman,Merman Fighter,Merman Hunter,Mermaid Initiate,Dune Burner,Thug
+            {TRAIT_RESILIENT}
+        [/trait_extra]
+        [trait_extra]
+            types=Heavy Infantryman,Bowman,Skeleton,Skeleton Archer,Saurian Augur,Troll Whelp,Orcish Archer
+            {TRAIT_FEARLESS}
+        [/trait_extra]
+        # This filters the following hero types
+        # to simply explain this...
+        # if your bonus point is next to mountains and cave walls and such, then these should be far away from
+        # watery bodies..thus, aquatic heroes should not be spawned in that case
+        [hero_spawn_filter]
+            types=Naga Fighter,Merman Fighter,Merman Hunter,Mermaid Initiate,Naga Dirkfang,Naga Guard
+            [filter_location]
+                [filter_radius]
+                    [not]
+                        terrain="M*,X*"
+                    [/not]
+                [/filter_radius]
+                terrain="W*,S*"
+                radius=2
+            [/filter_location]
+        [/hero_spawn_filter]
+    [/world_conquest_data]
+[/resource]
+
+[era]
+    id=extended_conquest_era
+    name= _ "Extended Conquest Era"
+    require_era=yes
+
+    description= _ "This is a string"
+
+    {RANDOM_SIDE}
+
+    # This is what an MP side code should look like 
+    [multiplayer_side]
+        id= # ID of the faction
+        name= # name of the faction
+        recruit=Drake Fighter,Dwarvish Fighter,Drake Burner,Dwarvish Thunderer,Saurian Augur,Dwarvish Ulfserker,Drake Clasher,Dwarvish Guardsman,Saurian Skirmisher,Poacher,Drake Glider,Footpad,Thief,Thief
+        # recruit list. For your own benefit, write the recruit list with pairs side by side
+        #as not to be confused later on
```

Comment:
Put these above the `recruit=` line, and maybe add a "so" in "side by side so as".